### PR TITLE
zh-cn translation of 66 Moneropedia entries

### PR DESCRIPTION
**Pull request by erciccione**
Originally opened on Gitlab: https://repo.getmonero.org/monero-project/monero-site/-/merge_requests/1121

This PR has been opened for months. It contains the translation of 66 moneropedia entries. Until now, nobody volunteered for the tasks. Understandable, since it's a huge amount of work to review, but we cannot merge it until somebody review at least part of it.

I will take care of rebasing it and make it ready to merge (was committed 7 months ago and never rebased), but we need a reviewer.

### CODE REVIEW

**Comment by mofadashi** [comment]
I would like to review it, but where I can find the original documentary?

**Comment by erciccione** [comment]
That would be great :) the original files are here: https://github.com/monero-project/monero-site/tree/master/_i18n/en/resources/moneropedia

Consider that some of the english files have been updated in the meantime, so the content could be slightly different. Would be ideal to have the missing parts translated as well.
Feel free to reach out if you need help.

File: _i18n/zh-cn/resources/moneropedia/account.md [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+
+如上例所示，将这些助记词存储在安全的位置非常重要。如果你担心在家里有重大损失的风险，您可以把种子的第二份副本存放在你的律师那里，或者放在一个保险箱内。我们也建议您用一种不要让人看出它是你的助记种子的储存方式，所以把它写进一封信或作为别的笔记的一部分是明智的。
+
+### 养成良好的安全习惯
+
+除了备份您的@助记种子以便在关键数据丢失时访问您的帐户之外，养成良好的安全习惯也很重要。在创建本地门罗币帐户时，使用一个安全性高的密码(不要用[MyMonero](https://mymonero.com)或其他基于网络的帐户系统)。
+
+永远不要把你的门罗币帐号密码给任何人，因为这可以用来在不知道你的@助记种子的情况下使用你电脑上的门罗币。同样，要确保你的杀毒软件运行良好，而且是最新的，尤其是在Windows系统的电脑上。
+
+最后，在点击电子邮件或不知名、不可信网站的链接时要格外小心，因为安装在你电脑上的恶意软件可能会在你调用资金之前，等着你进入你的门罗币账户。
```
Comment (mofadashi) [inline _i18n/zh-cn/resources/moneropedia/account.md:76]: “因为安装在你电脑上的恶意软件可能会在你调用资金之前，等着你进入你的门罗币账户。” -->因为安装在你电脑上的恶意软件会对你的账户虎视眈眈，在你进入账户后拿走你的钱

**Comment by erciccione** [comment]
@mofadashi I don't speak chinese, so i cannot really understand the context of your review. Could you use [GitHub suggestions](https://help.github.com/en/github/collaborating-with-issues-and-pull-requests/reviewing-proposed-changes-in-a-pull-request) when suggesting changes, so that i can jsut apply that.

Also, you don't need to review [the files listed here](https://github.com/monero-project/monero-site/issues/910#issuecomment-628562953), i will remove them soon from the repository and this PR.

**Review by mofadashi — CHANGES_REQUESTED** [review]
Made some changes during the review

File: _i18n/zh-cn/resources/moneropedia/account.md [hunk 2]
```diff
@@ -0,0 +1,80 @@
+---
+terms: ["account", "accounts", "wallet", "wallets"]
+summary: "similar in function to a bank account, contains all of your sent and received transactions"
+---
+
+### 基础知识
+
+那些熟悉门罗币的前辈（例如：比特币）的人会更熟悉“钱包”这个术语，用以描述这些。在门罗币这里，我们称之为一个帐户，它是一个由门罗币用户拥有和操作的私人帐户。
+
+您的帐户包含您发送和接收的所有门罗币@交易。你的账户余额是你收到的所有门罗币的总和，减去你发送的门罗币。当使用门罗币时，您可能会注意到您的帐户有两个余额，一个锁定的和一个非锁定的余额。未锁定余额包含可以立即使用的资金，而锁定余额包含不能立即使用的资金。您可能收到一个设置了@解锁时间的交易，或者您可能发送了一些门罗币并等待@找零回到您的钱包，这两种情况都可能导致这些资金被锁定一段时间。
```
Comment (mofadashi) [inline _i18n/zh-cn/resources/moneropedia/account.md:10]:
```suggestion
您的帐户包含您发送和接收的所有门罗币@交易。你的账户余额是你收到的所有门罗币减去你发送的门罗币。当使用门罗币时，您可能会注意到您的帐户有两个余额，一个锁定的和一个未锁定的余额。未锁定余额包含可以立即使用的资金，而锁定余额包含不能立即使用的资金。您可能收到一个设置了@解锁时间的交易，或者您可能发送了一些门罗币并等待@找零回到您的钱包，这两种情况都可能导致这些资金被锁定一段时间。
```

File: _i18n/zh-cn/resources/moneropedia/account.md [hunk 3]
```diff
@@ -0,0 +1,80 @@
...
+创建一个门罗币帐户无需任何费用，除了@矿工收取的个人@交易费用之外，不收取任何其他费用。
+
+
+### 加密密钥
+
+门罗币很依赖于一种名为*公/私钥加密* 或 *非对称密码学* 的密码学原理，这在[维基百科文章](https://en.wikipedia.org/wiki/publickey_cryptography)中有详细的描述。
+
+您的帐户基于两个密钥，一个@支付密钥和一个@查看密钥。@支付密钥的特殊之处在于，它是花费门罗币资金所需的唯一密钥，而@查看密钥允许您向第三方披露@交易记录，例如用于审计或会计目的。您帐户中的这些密钥在门罗币的@交易隐私当中，也扮演着重要的角色。
+
+为了保护您的帐户隐私，您必须保护这两个密钥。另一方面，公钥显然是公开的(它们是门罗币帐户地址的一部分)。对于普通的公/私钥加密，有人可以通过使用您的任何一个公钥加密来发送一条私钥消息，然后您将是惟一能够使用您的私钥解密它的人。
```
Comment (mofadashi) [inline _i18n/zh-cn/resources/moneropedia/account.md:25]:
```suggestion
为了保护您的帐户隐私，您必须保护这两个密钥。另一方面，公钥显然是公开的(它们是门罗币帐户地址的一部分)。对于普通的公/私钥加密，有人可以通过使用您的任何一个公钥加密来发送一条私钥消息，而您将是惟一能够使用您的私钥解密它的人。
```

File: _i18n/zh-cn/resources/moneropedia/account.md [hunk 4]
```diff
@@ -0,0 +1,80 @@
...
+
+### 备份您的帐户
+
+当您使用私有的@支付密钥管理您自己的门罗币帐户时，您将全权负责您资金的安全性。谢天谢地，门罗币让您的帐户备份变得非常容易。当您第一次创建一个门罗币帐号时，您的帐号会有一个独一无二的@助记种子，由你选择的语言的13或25个单词组成。**该种子是您唯一需要为您的帐号备份的东西**，因此必须将其记录下来并安全地存储。永远不要把种子储存在别人可以看到的地方!
```
Comment (mofadashi) [inline _i18n/zh-cn/resources/moneropedia/account.md:29]:
```suggestion
当您使用私有的@支付密钥管理您自己的门罗币帐户时，您将全权负责您资金的安全性。谢天谢地，门罗币让您的帐户备份变得非常容易。当您第一次创建一个门罗币帐号时，您的帐号会有一个独一无二的@助记种子，由你选择的语言的13或25个单词组成。**该种子是您帐号备份唯一需要东西**，因此必须将其记录下来并安全地存储。永远不要把种子储存在别人可以看到的地方!
```

**Comment by mofadashi** [comment]
> @mofadashi I don't speak chinese, so i cannot really understand the context of your review. Could you use GitHub suggestions when suggesting changes, so that i can jsut apply that.
> Also, you don't need to review the files listed here, i will remove them soon from the repository and this PR.

I finished my suggestion. could you take a look at it? Merge it?

**Comment by erciccione** [comment]
@mofadashi SInce this is not urgent i'm giving other translators time to review your suggestions. If nobody shows up i'll just add them.

**Comment by erciccione** [comment]
Added @mofadashi's review, fixed some minor technical issues and rebased

**Pull request merged**